src/c/core/session/session_info.c

```diff
@@ -70,15 +65,10 @@ void uxr_buffer_delete_session(uxrSessionInfo* info, ucdrBuffer* ub)
 
 void uxr_read_create_session_status(uxrSessionInfo* info, ucdrBuffer* ub)
 {
+    (void) info;
     STATUS_AGENT_Payload payload;
     (void) uxr_deserialize_STATUS_AGENT_Payload(ub, &payload);
-
-    if(UXR_REQUEST_LOGIN == info->last_request_id)
-    {
-        uxrObjectId object_id; uint16_t request_id;
-        uxr_parse_base_object_request(&payload.base.related_request, &object_id, &request_id);
-        process_create_session_status(info, payload.base.result.status, request_id, &payload.agent_info);
```

Comment:
The result status and the agent_info continuous arriving in the agent. It should be treated?